integrations/kubecost-aws-distro-open-telemetry.md

```diff
@@ -0,0 +1,113 @@
+# # Kubecost with AWS Distro for OpenTelemetry (ADOT)
+
+## Overview
+
+This guide will walk you through the steps to deploy Kubecost with [AWS Distro for Open Telemetry (ADOT)](https://aws-otel.github.io/) to collect metrics from your Kubernetes cluster.
+
+![Architecture Diagram](/images/adot-architecture.png)
+
+## Prerequisites
+
+Before following this guide, make sure you've reviewed AWS' doc on [installing the ADOT daemonset](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-onboard-ingest-metrics-OpenTelemetry.html). Update all configuration files in this folder that contain `YOUR_*` with your values.
+
+This guide assumes that the Kubecost Helm release name and the Kubecost namespace have the same value (usually this will be `kubecost`), which allows a global find and replace on `YOUR_NAMESPACE`.
+
+## Configuration
+
+Update all configuration files with your cluster name (replace all instances of `YOUR_CLUSTER_NAME_HERE`). The examples use `cluster` for the cluster name. You can use any key you want, but you will need to update the ConfigMap and deployment files to match. A simplified version of the ADOT DS installation is below.
+
+### ADOT configuration
+
+There are many options for deploying the ADOT daemonSet. At a minimum, Kubecost needs the provided scrape config to be added to the ADOT Prometheus ConfigMap. This sample ConfigMap also contains cAdvisor metrics, which is required by Kubecost.
+
+```bash
+kubectl apply -f example-configs/prometheus-daemonset.yaml
+```
+
+Alternatively, you can add these items to your [existing ConfigMap](https://github.com/kubecost/poc-common-configurations/blob/main/aws-amp/adot/example-configs/kubecost-adot-scrape-config.yaml).

```

Comment:
A lot of the files in this doc (e.g. `prometheus-damonset.yaml`, `iam-read-amp-metrics.json`, `iam-kubecost-metrics-s3-policy.json`) are coming from the poc-common-configs repo. Should we make that more clear?